api-inbound.yaml

```diff
@@ -100,14 +100,34 @@ components:
     ScanNotification:
       type: object
       required:
+        - scanID
```

Comment:
Question: Since when do we use this endpoint? And are you sure we need to take in startTime, endTime, and scanType in the ingress? These fields are what scan-notifier produces

Reply:
So, scan-notifier is somewhat unique in that it is a hybrid approach, following both the producer pattern, and the standard HTTP request-response model. It produces these ScanNotifications via Benthos, but also returns the same data in the HTTP response.

We discard that data, but it still returns it. As one of our earliest services, this was a way of hedging our bets of which pattern was going to be ultimately adopted.